docs/DeveloperGuide.md

```diff
@@ -594,6 +606,79 @@ testers are expected to do more *exploratory* testing.
     3. Test case: `filter p/91234567`<br>
        Expected: Shows only exact phone number matches.
 
+### Viewing weddings
+
+Success action: When wedding is successfully viewed, the details of the viewed wedding is shown in the status message and reflected in wedding list.
+The persons involved in the viewed wedding will be shown in the person list. Timestamp in the status bar is updated.
+
+1. Viewing a wedding while all weddings are being shown
+
+    1. Prerequisites: List all weddings using the `list` command. Multiple weddings in the list.
+
+    1. Test case: `vieww 1` <br>
+       Expected: First wedding from the wedding list is viewed. Success action will be carried out for that wedding.
+
+    1. Test case: `vieww 0`<br>
+       Expected: No wedding is viewed. Error details shown in the status message.
+
+    1. Test case: `vieww x` (where x is larger than the size of wedding list)<br>
+       Expected: Error message prompting the user to choose an index within the range shown.
+
+    1. Other incorrect vieww commands to try: `vieww`, `vieww x` (where x is a negative integer)<br>
+       Expected: Similar to point #1(iii).
+
+    1. Test case: `vieww Alice` <br>
+       Expected (Unique Alice): The wedding of `Alice` will be shown. Success action will be carried out for that wedding.<br>
+       Expected (Duplicated Alice): Weddings with name field matching `Alice` exactly will be shown.
+       Message will be shown to prompt the user to specify which wedding of `Alice` they want to view.<br>
+       Expected (No Alice): No wedding is viewed. Error details is shown in the status message.
+
+1. Viewing a wedding while a filtered list of weddings is shown
```

Comment:
I am not quite sure what you mean by this

Reply:
It's for cases when say there are two weddings with the same name (e.g. Alice). If I `vieww anna`,  the filtered list of anna weddings will appear and we will need to specify the index. The index will be according to the filtered list of wedding shown. Similar to the normal view command.

Does that make sense? How should I phrase it to make it less confusing?